forge-gui/res/quest/precons/Draconic Domination.dck

```diff
@@ -0,0 +1,110 @@
+[metadata]
+Name=Draconic Domination
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
```

Comment:
So I'm noticing a number of these files have two different `[metadata]` lines... probably should delete the second one

Reply:
I will merge this now and then make a PR to correct these